docs/API.md

```diff
@@ -6,6 +6,8 @@ These are best practices related to the current API used for App. This does not
 - There should be no client-side handling that is unique to any API call.
 - Data is pushed to the client and put straight into Onyx by low-level libraries.
 - Clients should be kept up-to-date with many small incremental changes to data.
+- Creating data needs to be flawlessly fast on every connection (offline, slow 3G, etc).
```

Comment:
https://expensify.slack.com/archives/C02HWMSMZEC/p1655240108188069?thread_ts=1654882263.720009&cid=C02HWMSMZEC

Reply:
Suggestion: Remove the word "flawlessly". I like the sentiment, but I think it's an unrealistic expectation.

Reply:
@aldo-expensify yeah, good point, I rephrased this point and added an example, thanks!